src/lib.rs

```diff
@@ -350,23 +330,25 @@ fn view_footer(todos: &BTreeMap<Ulid, Todo>, selected_filter: Filter, base_url:
     ]
 }
 
-fn view_filters(selected_filter: Filter, base_url: &Url) -> Node<Msg> {
+fn view_filters() -> Node<Msg> {
     ul![
         C!["filters"],
-        Filter::iter().map(|filter| {
-            let urls = Urls::new(base_url);
-
-            let (url, title) = match filter {
-                Filter::All => (urls.home(), "All"),
-                Filter::Active => (urls.active(), "Active"),
-                Filter::Completed => (urls.completed(), "Completed"),
+        Route::iter().filter_map(|route| {
+            let title = match route {
+                Route::All => Some("All"),
+                Route::Active => Some("Active"),
+                Route::Completed => Some("Completed"),
+                Route::NotFound => None, // or `return None`
```

Comment:
I think `return None` would make the code much cleaner.

Reply:
ok :+1: